.github/workflows/ci.yml

```diff
@@ -465,6 +466,16 @@ jobs:
           name: docker-image-prometheus-exporter-file
           path: ./tmp
       - run: docker load --input ./tmp/image*.tar
+      - name: Log in to GitHub registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Retag all images in order to be pushed to GHCR.io
+        run: docker image ls --format "{{.Tag}}" | grep prometheus | xargs -I % docker tag $DOCKER_IMAGE:% $DOCKER_GH_IMAGE:%
```

Comment:
This is risky as more then our images might be returning from that `docker images` call. There should be a tags file in `./tmp`

Reply:
I refactored the code trying to follow your suggestion, have a look from line 479 to 484

https://github.com/usabilla/php-docker-template/pull/189/commits/0d985e03ec70cd614005c5c0f32ee7472bbddf6c

If this is all right I'll use the same pattern also for push-php/push-http